File: Kattis/C++/Fizz-Buzz/Fizz-Buzz.cpp
#include <bits/stdc++.h>

using namespace std;
typedef int64_t i64;
using d6 = long double;

int main()
{
    i64 x,y,n;

    cin >> x >> y >> n;
    
    for(i64 i = 1;i <= n; i++)
    {
        if(i % x == 0 && i % y == 0)
        {
            cout << "FizzBuzz" << endl;
        }
        else if(i % x ==0)
        {
            cout << "Fizz" << endl;
        }
        else if(i % y == 0)
        {
            cout << "Buzz" << endl;
        }
        else
        {
            cout << i << endl;
        }
    }

}

File: Kattis/Python/FizzBuzz.py

def main():
    x,y,n = input().split()

    x = int(x)
    y = int(y)
    n = int(n)

    for i in range(1,n+1):
        if i % x == 0 and i % y == 0:
            print("FizzBuzz", end="\n")
        elif i % x == 0:
            print("Fizz", end="\n")
        elif i % y == 0:
            print("Buzz", end="\n")
        else:
            print(i, end="\n")

if __name__ == "__main__":
    main()